app/app.go

```diff
@@ -11,8 +11,9 @@ import (
 )
 
 type App struct {
-	Srv       *Server
-	PluginEnv *pluginenv.Environment
+	Srv                    *Server
+	PluginEnv              *pluginenv.Environment
+	PluginConfigListenerId string
```

Comment:
This might not be the best place for this but it's good enough for now to fix the tests